batchflow/models/torch/layers/core.py

```diff
@@ -156,22 +191,21 @@ def forward(self, x):
                     sizes = self.multisample
                 elif all(isinstance(item, float) for item in self.multisample):
                     if sum(self.multisample) != 1.:
-                        raise ValueError('Sequence of floats must sum up to one for multisample dropout,\
-                                          got instead {}'.format(self.multisample))
+                        raise ValueError(f'Sequence of floats must sum up to one, got {self.multisample} instead!')
 
                     batch_size = x.shape[0]
                     sizes = [round(batch_size*item) for item in self.multisample[:-1]]
                     residual = batch_size - sum(sizes)
                     sizes += [residual]
                 else:
-                    raise ValueError('Elements of multisample must be either all ints or floats,\
-                                      got instead {}'.format(self.multisample))
+                    raise ValueError(f'Elements of multisample must be either all ints or floats,\
+                                       got "{self.multisample}" instead!')
```

Comment:
```suggestion
                    raise ValueError('Elements of multisample must be either all ints or floats,'
                                       f' got "{self.multisample}" instead!')
```

will be prettier (without printed extra spaces)
![image](https://user-images.githubusercontent.com/21172482/168097184-fb9b2db5-f0e2-4da3-87e7-aff7799c82af.png)